MAINTAINERS.yml

```diff
@@ -0,0 +1,1052 @@
+
+"Drivers: IPM":
+    # Provisional entry, to be corrected
+    status: maintained
+    maintainers:
+        - andrewboie
+    files:
+        - drivers/ipm/
+    description: >-
+        Inter-processor mailboxes
+
+"Drivers: I2C":
+    # Provisional entry, to be corrected
+    # TODO: spreadsheet not matching CODEOWNERS
+    status: maintained
+    maintainers:
+        - mbolivar
+    files:
+        - drivers/i2c/
+        - dts/bindings/i2c/
+        - include/i2c.h
+        - include/drivers/i2c.h
+        - include/drivers/i2c/
+    labels:
+        - "area: I2C"
+
+"Drivers: I2S":
+    # Provisional entry, to be corrected
+    status: orphaned
+    files:
+        - doc/reference/peripherals/i2s.rst
+        - drivers/i2s/
+        - dts/bindings/i2s/
+        - include/i2s.h
+        - include/drivers/i2s.h
+        - tests/drivers/i2s/
+    labels:
+        - "area: I2S"
+
+"Drivers: LED":
+    # Provisional entry, to be corrected
+    status: orphaned
+    collaborators:
+        - Mani-Sadhasivam
+    files:
+        - drivers/led/
+        - include/led.h
+        - include/drivers/led/
+        - include/drivers/led.h
+        - samples/drivers/led_*/
+    labels:
+        - "area: LED"
+
+"Drivers: LED Strip":
+    # Provisional entry, to be corrected
+    status: orphaned
+    collaborators:
+        - mbolivar
+    files:
+        - drivers/led_strip/
+        - dts/bindings/led_strip/
+        - include/led_strip.h
+        - include/drivers/led_strip.h
+
+"Drivers: Modem":
+    # Provisional entry, to be corrected
+    status: maintained
+    maintainers:
+        - mike-scott
+    files:
+        - drivers/modem/
+    labels:
+        - "area: Modem"
+
+"Drivers: Neural networking":
+    # Provisional entry, to be corrected
+    status: orphaned
+    files:
+        - drivers/neural_net/
+
+"Drivers: PCI":
+    # Provisional entry, to be corrected
+    # TODO: spreadsheet not matching CODEOWNERS
+    status: maintained
+    maintainers:
+        - dcpleung
+    collaborators:
+        - andrewboie
+    files:
+        - drivers/pcie/
+        - include/drivers/pcie/
+    labels:
+        - "area: PCI"
+
+"Drivers: Pinmux":
+    # Provisional entry, to be corrected
+    status: maintained
+    maintainers:
+        - mnkp
+    files:
+        - doc/reference/peripherals/pinmux.rst
+        - drivers/pinmux/
+        - include/drivers/pinmux.h
+        - include/pinmux.h
+        - tests/drivers/pinmux/
+    labels:
+        - "area: Pinmux"
+
+"Drivers: PWM":
+    # Provisional entry, to be corrected
+    status: orphaned
+    files:
+        - drivers/pwm/
+        - dts/bindings/pwm/
+        - include/drivers/pwm.h
+        - include/pwm.h
+        - tests/drivers/pwm/
+    labels:
+        - "area: PWM"
+
+"Drivers: Serial/UART":
+    # Provisional entry, to be corrected
+    # TODO: spreadsheet not matching CODEOWNERS
+    status: maintained
+    maintainers:
+        - dcpleung
+    files:
+        - drivers/serial/
```

Comment:
Also `dts/bindings/serial/`

Reply:
Thanks @dcpleung. Note that there was an issue/inconsistency regarding this entry: the CODEOWNERS file does not seem to contain any line assigned to you for these file. 
But I take your feedback as you agreeing otherwise about this entry.

Reply:
I am happy to look after this component. :)